tests/integration/helpers/network.py

```diff
@@ -212,6 +212,7 @@ def _ensure_container(self):
             self._container = self._docker_client.containers.run('yandex/clickhouse-integration-helper',
                                                                  auto_remove=True,
                                                                  command=('sleep %s' % self.container_exit_timeout),
+                                                                 volumes={'/run/xtables.lock': {'bind': '/run/xtables.lock', 'mode': 'rw' }},
```

Comment:
Oh, I missed that fact that it uses separate container to execute iptables.

BTW you don't need `rw` for `flock`

```sh
$ docker run -v /tmp/foo.lock:/tmp/foo.lock:ro alpine flock /tmp/foo.lock echo foo
foo
```